src/components/Dashboard/DashboardMain.js

```diff
@@ -0,0 +1,187 @@
+/*eslint-disable */
+import React from 'react'
+import {
+  Main,
+  Header,
+  Button,
+  IconPlus,
+  Tag,
+  SidePanel,
+  Split,
+  DataView,
+  Box,
+  GU,
+  Layout,
+  textStyle,
+  DropDown,
+  IdentityBadge,
+} from '@aragon/ui'
+import '../../reset.css'
+import notasksfound from '../../assets/notasksfound.svg'
+import Card from '@aragon/ui/dist/Card'
+import agreement1 from '../../assets/agreement 1.svg'
+import agreement2 from '../../assets/agreement 2.svg'
+import active from '../../assets/active.svg'
+import inactive from '../../assets/inactive.svg'
+import totalcost from '../../assets/totalcost.svg'
+import ChartComponent from './ChartComponent'
+import ChartComponentNew from './ChartComponentNew'
+var green = '#219653'
+var orange = '#FF7A3C'
+var primary = '#52006F'
+
+function DashboardMain() {
+  return (
+    <>
+      <div
+        css={`
+          ${textStyle('title1')};
+          margin-top: 0rem;
+          margin-left: 1rem;
+          margin-right: 1rem;
+        `}
+      >
+        Dashboard
+      </div>
+
+      <div
+        style={{
```

Comment:
Shall we maintain most of the CSS separately?

Reply:
Yes, perhaps we have to go in that direction eventually.